fn/lambda.go

```diff
@@ -214,9 +110,13 @@ func awsImport(c *cli.Context) error {
 		return err
 	}
 
+	if *function.Configuration.Runtime == "nodejs4.3" {
+		*function.Configuration.Runtime = "node"
```

Comment:
that would require updating the names of the docker images first for this PR to work